static/lang/en.json

```diff
@@ -3483,6 +3484,7 @@
         "TraitCapacity4": "Capacity 4",
         "TraitCapacity5": "Capacity 5",
         "TraitCarried": "Carried",
+        "TraitCavalry": "Cavalry",
```

Comment:
So close!

Reply:
I am so bad with alphabetical order you have no idea, I have to sing the childhood song each and every time.